packages/blockchain-wallet-v4-frontend/src/scenes/Settings/General/LinkedBanks/template.success.tsx

```diff
@@ -97,7 +169,7 @@ const Success: React.FC<InjectedFormProps<{}, Props> & Props> = (props) => {
                   >
                     <FormattedMessage id='buttons.remove' defaultMessage='Remove' />
                   </RemoveButton>
-                </Child>
+                </Child> */}
```

Comment:
Is this commented code ok to be commented?

Reply:
we should not commit commented code